README.md

```diff
@@ -4,71 +4,74 @@ This sample is currently work in progress.
 
 In this sample, a Visual Studio Code dev container is used as the developer sandbox to leverage a local K3D cluster, initialize Azure resources and install a baseline set of Azure IoT Operations' components.
 
-The objective is to illustrate how to setup observability to be able to locally vizualize health of the cluster and Azure IoT Operations' components. This repository is particularly useful for those interested in edge-based visualization. More details about the setup can be found [here](./docs/observability-setup.md).
+The objective is to illustrate how to setup observability to be able to locally visualize health of the cluster and Azure IoT Operations' components. This repository is particularly useful for those interested in edge-based visualization. More details about the setup can be found [here](./docs/observability-setup.md).
 
-If you’re looking for a default observability setup and wish to analyze observability data using Azure Monitor, follow [this documentation](https://learn.microsoft.com/en-us/azure/iot-operations/monitor/howto-configure-observability).
+If you’re looking for a default observability setup and wish to analyze observability data using Azure Arc extensions and Azure Monitor in the cloud, follow [this documentation](https://learn.microsoft.com/en-us/azure/iot-operations/monitor/howto-configure-observability).
 
 > [!WARNING]
-Azure IoT Operations is still in Preview. Any parts of sample could stop working as the product evolves towards General Availability. The current version is pinned in the dev container, and is Azure IoT Operations Preview v0.6.0.
+> Azure IoT Operations is still in Preview. Any parts of sample could stop working as the product evolves towards General Availability. The current version is pinned in this dev container is Azure IoT Operations Preview v0.8.0.
```

Comment:
maybe use the real version number v0.8.32...just my 2 cents

Reply:
Correct, updated